mon-pix/translations/fr.json

```
@@ -289,9 +289,18 @@
           "guidance": "'<p>'* Soyez attentifs à ce que vous écrivez dans cette zone : restez objectif et factuel pour votre intérêt et celui des autres.'</p><p>'Notamment, ne saisissez aucune information, vous concernant ou concernant des tiers, sur la santé, la religion, les opinions politiques, syndicales et philosophiques, les origines ethniques, ainsi que sur les sanctions et condamnations.'</p>'"
         }
       },
+      "illustration": {
+        "placeholder": "Chargement de l'image en cours"
+      },
       "progress-bar": {
         "label": "Question"
       },
+      "statement": {
+        "default-type": "Pix vous laisse le choix du format de fichier à télécharger. Si vous ne savez pas quelle option retenir, conservez le choix par défaut. Il correspond au format de fichier pris en charge par le plus grand nombre de logiciels.",
+        "action": "Télécharger",
+        "file": "fichier .{fileExtension}",
+        "file-type": "Choisissez le type de fichier que vous voulez utiliser"
+      },
```

Toutes ces clés concernent un téléchargement de fichier
```suggestion
"statement": {
  "file-download": {
    "actions": {
      "description": "Choisissez le type de fichier que vous voulez utiliser"
      "label": "Télécharger"
    }
    "description": "Pix vous laisse le choix du format de fichier à télécharger. Si vous ne savez pas quelle option retenir, conservez le choix par défaut. Il correspond au format de fichier pris en charge par le plus grand nombre de logiciels.",
     "file-type": "fichier .{fileExtension}"
  }
},
```

---

Comme il s'agit d'actions, ne faudrait-il pas utiliser des verbes d'action ?

```suggestion
    "actions": {
      "choose-type": "Choisissez le type de fichier que vous voulez utiliser"
      "download": "Télécharger"
    }
```